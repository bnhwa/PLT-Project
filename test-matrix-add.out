FAILED
  ./test-matrix-add.exe failed on ./test-matrix-add.exe
