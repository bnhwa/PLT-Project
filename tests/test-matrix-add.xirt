num main() {
    xirtam m;
    xirtam p;
    m = [[1, 2], [3, 4], [5, 6]];
    p = [[-1, -1], [5, 3.1], [-2, -1.11365]];
    printm(matadd(m, p));
}