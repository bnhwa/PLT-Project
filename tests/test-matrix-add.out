 0.00 1.00
 8.00 7.10
 3.00 4.89